# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/svg/SVGTextContentElement.h
# Commit: b848ca90888e
# Full Hash: b848ca90888ee8e446f3d54b65c3ef68c71a1451
# Author: longsonr <longsonr@gmail.com>
# Date: 2019-06-20 22:06:31
# Regressor Bug: 1551030
# File Overlap Count: 2
# Description:
#   Bug 1551030 - Merge all SVGRect classes r=dholbert
# ==============================================================================

diff -r 20956a783e08 -r b848ca90888e dom/svg/SVGTextContentElement.h
--- a/dom/svg/SVGTextContentElement.h	Thu Jun 20 14:19:57 2019 +0300
+++ b/dom/svg/SVGTextContentElement.h	Thu Jun 20 07:03:54 2019 -0700
@@ -20,7 +20,7 @@
 namespace dom {
 
 struct DOMPointInit;
-class SVGIRect;
+class SVGRect;
 
 typedef SVGGraphicsElement SVGTextContentElementBase;
 
@@ -46,7 +46,7 @@
   already_AddRefed<nsISVGPoint> GetEndPositionOfChar(uint32_t charnum,
                                                      ErrorResult& rv);
   MOZ_CAN_RUN_SCRIPT
-  already_AddRefed<SVGIRect> GetExtentOfChar(uint32_t charnum, ErrorResult& rv);
+  already_AddRefed<SVGRect> GetExtentOfChar(uint32_t charnum, ErrorResult& rv);
   MOZ_CAN_RUN_SCRIPT float GetRotationOfChar(uint32_t charnum, ErrorResult& rv);
   MOZ_CAN_RUN_SCRIPT int32_t GetCharNumAtPosition(const DOMPointInit& aPoint);
 